cross/redis/patches/002-lua-env.patch

```diff
@@ -12,7 +12,7 @@
 -AR= ar rcu
 -RANLIB= ranlib
 +CFLAGS+= -O2 -Wall $(MYCFLAGS)
-+AR:= $(AR) rcu
++AR= $(AR) rcu
```

Comment:
your chanage gives the following error output:
`Makefile:12: *** Recursive variable 'AR' references itself (eventually).  Stop.`

Your change should be reverted here.
The := operator is used to assign the AR variable the incomming AR as defined by the spksrc-framework and append  `" rcu"`.

Reply:
Indeed, reverted to comply with this rule.